mitosheet/mitosheet/step_performers/set_cell_value.py

```diff
@@ -135,10 +136,6 @@ def transpile( # type: ignore
         # We don't need to wrap the value in " if its None, a Boolean Series, or a Number Series.
         if type_corrected_new_value is None or is_bool_dtype(column_dtype) or is_number_dtype(column_dtype):
             code.append(f'{post_state.df_names[sheet_index]}.at[{row_index}, {transpiled_column_header}] = {type_corrected_new_value}')
-        elif column_mito_type == DATETIME_SERIES:
-            code.append(f'{post_state.df_names[sheet_index]}.at[{row_index}, {transpiled_column_header}] = pd.to_datetime(\"{type_corrected_new_value}\")')
-        elif column_mito_type == TIMEDELTA_SERIES:
-            code.append(f'{post_state.df_names[sheet_index]}.at[{row_index}, {transpiled_column_header}] = pd.to_timedelta(\"{type_corrected_new_value}\")')
```

Comment:
@aarondr77 you need to keep these boys! Instead of deleting them, we need to just use the `is_datetime_dtype` checks, etc.